internal/wasmdebug/dwarf.go

```diff
@@ -137,44 +137,68 @@ entry:
 		return
 	}
 
+	ln := lines[index]
+	if ln.addr != instructionOffset {
+		// If the address doesn't match exactly, the previous entry is the one that contains the instruction.
+		if index-1 < 0 {
```

Comment:
maybe add a "for example" here with a high level of how something doesn't match exactly (don't need to paste the trace, just mention high level about the rust maybe with inlining which can cause this)

Reply:
https://github.com/tetratelabs/wazero/pull/916/commits/cee9b05123a2cb22ad8d06548e985c8a1d81f461 added comment